README.md

```diff
@@ -22,7 +22,7 @@ ARG SSL=true
 ARG MP_MONITORING=true
 
 # This script will add the requested XML snippets and grow image to be fit-for-purpose
-RUN /configure.sh
+RUN /opt/ibm/helpers/build/configure.sh
```

Comment:
I had added `/opt/ibm/helpers/build` to the `PATH` of the container so that we don't need to specify the full path (which is easier for customers).  Do we really need the full path here?

Reply:
https://github.com/WASdev/ci.docker/blob/master/ga/18.0.0.4/kernel/Dockerfile#L35

Reply:
Short path is good and convenient to users.

When I tested locally, `RUN /configure.sh` didn't work, hence used full path. The slash at the front was the problem. Just tried and `RUN configure.sh` works fine. Made the changes.